lib/main.js

```diff
@@ -322,9 +322,14 @@ if (!process.env.CI) {
     })
   })
 
+  // TODO: These should be tested as provisioning variants
+  // Allow the user to set image maker configuration options as env vars.
```

Comment:
The comment is strictly about the TS4900 device (it doesn't make sense for a BeagleBone), so I think it should remain where it was.